Updated runtime installer name to WindowsAppSDKInstall.exe

qmatteoq opened the pull request:
The documentation still refers to ReunionRuntimeInstaller.exe, but the version currently published on GitHub has already been rebranded to WindowsAppSDKInstall.exe

qmatteoq committed:
Updated runtime installer name to WindowsAppSDKInstall.exe

The documentation still refers to ReunionRuntimeInstaller.exe, but the version currently published on GitHub has already been rebranded to WindowsAppSDKInstall.exe

PRMerger13 commented:
@qmatteoq : Thanks for your contribution! The author(s) have been notified to review your proposed change.

qmatteoq committed:
Updated name of the Bootstrap library

In the latest release of the runtime, the bootstrap library has been renamed from Microsoft.ProjectReunion.Bootstrap.dll to Microsoft.WindowsAppSDK.Bootstrap.dll

PRMerger17 commented:
@qmatteoq : Thanks for your contribution! The author(s) have been notified to review your proposed change.

ktoliver commented:
The content has been moved or reconfigured since the pull request was opened. The pull request can't be merged in, so we'll need to close it. If the proposed update still needs to be added to the content, please submit it in a new pull request. Thanks.
